index.html

```diff
@@ -9,31 +9,32 @@
     <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet"/>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="style.css" />
+    <script src="app.js" defer></script>
   </head>
   <body>
     <header class="head">
       <div class="header container">
         <div class="logo">
-          <h3>Dave Mba</h3>
+          <h3>Dave Mba.</h3>
         </div>
         <div class="nav">
           <nav>
+            <i class="fa-solid fa-xmark xmark"></i>
             <ul>
-              <li><a href="">Portfolio</a></li>
-              <li><a href="">About</a></li>
-              <li><a href=""> Contact</a></li>
+              <li><a href="#portfolio" class="nav-link">Portfolio</a></li>
```

Comment:
- [ ] Please know that the portfolio link should take the user to the `recent works` sections, kindly fix this. 😁